build.gradle.kts

```diff
@@ -17,4 +17,21 @@
 plugins {
     id("eclipse")
     id("idea")
+    alias(libs.plugins.nexus.publish)
```

Comment:
Curious why all this can't be in `publishing.gradle.kts`.  Can you explain?

Reply:
Its hidden a little in the second commit message:

> As the nexus-publish plugin can only be configured in the root project
the username and password logic was moved out of conventions/publishing.gradle.kts
and into the root buid.gradle.kts.
> 
> Note: The nexus-publish plugin builds upon the gradle maven-publish
plugin so even though the configuration is in the root project,
it will only publish projects that use the publishing convention.